### PR TITLE
Require manual start signal for the replayer for CDK

### PR DESCRIPTION
**Pull request by lewijacn**
### Description
Sets the default traffic replayer number of instances to 0 and provides basic documentation for how a user should start the replayer.

Includes minor fixes from Brian's experience with deployment

Includes security fix for babel NPM

### Issues Resolved
N/A

### Testing
Deployment testing

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts [hunk 1]
```diff
@@ -144,7 +144,7 @@ export class MigrationServiceCore extends Stack {
             cluster: ecsCluster,
             taskDefinition: serviceTaskDef,
             assignPublicIp: true,
-            desiredCount: props.taskInstanceCount ? props.taskInstanceCount : 1,
+            desiredCount: props.taskInstanceCount,
```
Comment (gregschohn) [inline deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts:147]: Will we be automatically starting the metrics collection container?

File: deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts [hunk 2]
```diff
@@ -76,14 +76,21 @@ export class MigrationConsoleStack extends MigrationServiceCore {
                 "kafka-cluster:*"
             ]
         })
-
+        const allReplayerServiceArn = `arn:aws:ecs:${props.env?.region}:${props.env?.account}:service/migration-${props.stage}-ecs-cluster/migration-${props.stage}-traffic-replayer*`
+        const updateReplayerServicePolicy = new PolicyStatement({
+            effect: Effect.ALLOW,
+            resources: [allReplayerServiceArn],
+            actions: [
+                "ecs:UpdateService"
```
Comment (gregschohn) [inline deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts:84]: This reads weird.  The replayers can update themselves?

File: deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts (hunk 1 above)
Comment (lewijacn) [reply deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts:147]: Yes that is the default. Is there a problem there if the Replayer isn't active?

File: deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts (hunk 2 above)
Comment (lewijacn) [reply deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts:84]: Only the migration console can update the replayers

File: deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts (hunk 1 above)
Comment (gregschohn) [reply deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts:147]: sorry - I read this backward for some reason.

**Review by gregschohn — APPROVED** [review]

**Pull request merged**